packages/features/bookings/Booker/components/hooks/useVerifyEmail.ts

```diff
@@ -32,6 +36,11 @@ export const useVerifyEmail = ({
     },
   });
 
+  const { data: userWithEmail } = trpc.viewer.public.userWithEmail.useQuery({
```

Comment:
NIT: change the name of variable `userWithEmail`.

Reply:
Yeah agreed. Let's change that. It doesn't really tell you what it is accomplishing.